CHANGELOG.rst

```diff
@@ -0,0 +1,8 @@
+=========
+Changelog
+=========
+
+Version 0.2dev
```

Comment:
Given the release we just made, we should probably separate this out into a `Version 0.1.1`, which is easily installable, and then have an `Unreleased version` that turns into `0.2.0` when we've got tests going.

Reply:
Agreed. Fixed.